### PR TITLE
Fixes #19884 - Disable Style/NumericPredicate cop

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: app/models/concerns/compute_resource_console_common.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module ComputeResourceConsoleCommon
   extend ActiveSupport::Concern
 
   def set_console_password?
-    !(attrs[:setpw] == 0) # return true unless attrs[:setpw] is set to 0
+    !(attrs[:setpw].nil? || attrs[:setpw].zero?) # return true unless attrs[:setpw] is set to nil or 0
```
Comment (tbrisker) [inline app/models/concerns/compute_resource_console_common.rb]: this is a logic change, since 
```ruby 
nil == 0  # ==> false
```

**Review by tbrisker — COMMENTED** [review]
I'm not sure how i feel about this cop. I don't feel like `.positive?` is necessarily better than `> 0`. Specifically, enforcing `.zero?` can potentially lead to errors since `nil == 0` is false and `nil.zero?` is undefined (which isn't an issue for `>` and `<` since they don't work with `nil`)

**Comment by ares** [comment]
I'd prefer to disable this one

**Comment by mmoll** [comment]
Disabled this cop.

**Review by tbrisker — APPROVED** [review]
Thanks @mmoll !

**Pull request merged**